### ISSUE TITLE
failing to install the packages sudo apt-get install libnvinfer6 libnvonnxparsers libnvparsers6 libnvinfer-

### ISSUE THREAD

**Issue by nagaraj-trivedi** [NVIDIA/TensorRT#3290]
## Description
I ran the following instructions in a cell in jupyter notebook and it is failing to install 
sudo apt-get install libnvinfer6=${version} libnvonnxparsers6=${version} libnvparsers6=${version} libnvinfer-

I am executing the code given in this notebook in google coallb. Below are the statements run in the cell

%%bash

version="6.0.1-1+cuda10.1"
wget https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64/nvidia-machine-learning-repo-ubuntu1804_1.0.0-1_amd64.deb

dpkg -i nvidia-machine-learning-repo-*.deb
apt-get update

sudo apt-get install libnvinfer6=${version} libnvonnxparsers6=${version} libnvparsers6=${version} libnvinfer-

## Environment

Google collab

**TensorRT Version**:  -NA-

**NVIDIA GPU**: -NA-

**NVIDIA Driver Version**: -NA-

**CUDA Version**: -NA-

**CUDNN Version**: -NA-

Operating System: -NA-

Python Version (if applicable): -NA-

Tensorflow Version (if applicable): -NA-

PyTorch Version (if applicable): -NA-

Baremetal or Container (if so, version): -NA-

## Relevant Files

-NA-

**Model link**:
https://github.com/NVIDIA/TensorRT/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=XXX+failure+of+TensorRT+X.Y+when+running+XXX+on+GPU+XXX

## Steps To Reproduce
1. Copy the instructions from  the notebook from this git hub link  and execute them in the google collab jupyter notebook
2. Execute below instruction in a cell
. %%bash

version="6.0.1-1+cuda10.1"
wget https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64/nvidia-machine-learning-repo-ubuntu1804_1.0.0-1_amd64.deb

dpkg -i nvidia-machine-learning-repo-*.deb
apt-get update

sudo apt-get install libnvinfer6=${version} libnvonnxparsers6=${version} libnvparsers6=${version} libnvinfer-

**Commands or scripts**: -NA-

**Have you tried [the latest release](https://developer.nvidia.com/tensorrt)?**: -NA-

**Can this model run on other frameworks?** For example run ONNX model with ONNXRuntime (`polygraphy run <model.onnx> --onnxrt`): -NA-

**Comment by zerollzeng** [comment]
6.0 is very old. Could you please use the latest release?

**Comment by zerollzeng** [comment]
I think `pip3 install tensorrt` should just work for you.

**Comment by zerollzeng** [comment]
One think is I also don't know whether we support install on colab officially, I'll let @rajeevsrao to answer this question.

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, pls reopen if you still have question, thanks!
